src/components/Tool/Home/ToolList.ts

```diff
@@ -0,0 +1,82 @@
+import {
+  mdiTrophy,
+  mdiGoogle,
+  mdiTable,
+  mdiCloudUpload,
+  mdiCurrencyUsd,
+  mdiEmail,
+  mdiMap,
+  mdiPhone,
+  mdiNewspaperVariantOutline,
+  mdiHome,
+  mdiAccountGroup,
+} from '@mdi/js';
+
+export const ToolsList = [
+  {
+    tool: 'Appeal',
```

Comment:
Can you translate these?
```ts
import i18n from 'i18next';
```
```suggestion
    tool: i18n.t('Appeal'),
```

If you're in a component (which you're not here, which is fine) you can use the hook:
```ts
import { useTranslation } from 'react-i18next';

export ExampleComponent = () => {
  const { t } = useTranslation();
  return <p>{t('Some English String')}</p>
}
```